### PR TITLE
Developer portfolio links

### PR DESCRIPTION
**Pull request by bjv753**

### CODE REVIEW

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -20,15 +20,18 @@ export default function Footer() {
                 
                 </div>
                 <div className="footer-links">
-                    {/* <p className="footer-title">Links</p>
-                    <a href="" className="website-link">www.website.com</a>
-                    <a href="" className="website-link">www.website.com</a> */}
+                    <p className="footer-title">Links</p><br />
+                    <h3>Developers: Dashona Holt, Lucile Jones, Natalie Taylor, Benjamin Vredevelt</h3> <br />
+                    <a href="https://dashonaholtsportfolio.netlify.app/" className="website-link">Dashona's Portfolio</a>
```
Comment (angelodalisay) [inline src/components/Footer.jsx:25]: add 
`target="_blank" `
to keep visitors within the site

File: src/components/Footer.jsx [hunk 2]
```diff
@@ -20,15 +20,18 @@ export default function Footer() {
                 
                 </div>
                 <div className="footer-links">
-                    {/* <p className="footer-title">Links</p>
-                    <a href="" className="website-link">www.website.com</a>
-                    <a href="" className="website-link">www.website.com</a> */}
+                    <p className="footer-title">Links</p><br />
+                    <h3>Developers: Dashona Holt, Lucile Jones, Natalie Taylor, Benjamin Vredevelt</h3> <br />
+                    <a href="https://dashonaholtsportfolio.netlify.app/" className="website-link">Dashona's Portfolio</a>
+                    <a href="https://lucilejones-portfolio.onrender.com/" className="website-link">Lucile's Portfolio</a>
```
Comment (angelodalisay) [inline src/components/Footer.jsx:26]: add 
`target="_blank" `
to keep visitors within the site

File: src/components/Footer.jsx [hunk 3]
```diff
@@ -20,15 +20,18 @@ export default function Footer() {
...
                 </div>
                 <div className="footer-links">
-                    {/* <p className="footer-title">Links</p>
-                    <a href="" className="website-link">www.website.com</a>
-                    <a href="" className="website-link">www.website.com</a> */}
+                    <p className="footer-title">Links</p><br />
+                    <h3>Developers: Dashona Holt, Lucile Jones, Natalie Taylor, Benjamin Vredevelt</h3> <br />
+                    <a href="https://dashonaholtsportfolio.netlify.app/" className="website-link">Dashona's Portfolio</a>
+                    <a href="https://lucilejones-portfolio.onrender.com/" className="website-link">Lucile's Portfolio</a>
+                    <a href="https://sisterchicago.github.io/portfolio/" className="website-link">Natalie's Portfolio</a>
```
Comment (angelodalisay) [inline src/components/Footer.jsx:27]: add 
`target="_blank" `
to keep visitors within the site

File: src/components/Footer.jsx [hunk 4]
```diff
@@ -20,15 +20,18 @@ export default function Footer() {
...
+                    <a href="https://super-stardust-4644df.netlify.app/about" className="website-link">Benjamin's Portfolio</a>
```
Comment (angelodalisay) [inline src/components/Footer.jsx:28]: add 
`target="_blank" `
to keep visitors within the site

File: src/components/Footer.jsx [hunk 5]
```diff
@@ -20,15 +20,18 @@ export default function Footer() {
                 
                 </div>
                 <div className="footer-links">
-                    {/* <p className="footer-title">Links</p>
-                    <a href="" className="website-link">www.website.com</a>
-                    <a href="" className="website-link">www.website.com</a> */}
+                    <p className="footer-title">Links</p><br />
+                    <h3>Developers: Dashona Holt, Lucile Jones, Natalie Taylor, Benjamin Vredevelt</h3> <br />
```
Comment (angelodalisay) [inline src/components/Footer.jsx:24]: Change verbiage from `Developers: Dashona Holt, Lucile Jones, Natalie Taylor, Benjamin Vredevelt`

to `Made By:`

**Review by angelodalisay — COMMENTED** [review]
please fix changes for pull request to be approved.

File: src/components/Footer.jsx (hunk 4 above)
Comment (bjv753) [reply src/components/Footer.jsx:28]: Cool! Just to confirm, I need to add target="_blank" as a property within the 'a' tag?

File: src/components/Footer.jsx (hunk 5 above)
Comment (bjv753) [reply src/components/Footer.jsx:24]: On it!

File: src/components/Footer.jsx (hunk 4 above)
Comment (angelodalisay) [reply src/components/Footer.jsx:28]: that is correct

Comment (bjv753) [reply src/components/Footer.jsx:28]: Sounds good! Thank you

File: src/components/Footer.jsx [hunk 6]
```diff
@@ -20,15 +20,17 @@ export default function Footer() {
                 
                 </div>
                 <div className="footer-links">
-                    {/* <p className="footer-title">Links</p>
-                    <a href="" className="website-link">www.website.com</a>
-                    <a href="" className="website-link">www.website.com</a> */}
+                    <h3>Made By:</h3><br />
```
Comment (angelodalisay) [inline src/components/Footer.jsx:23]: I removed full names for security purposes.

File: src/components/Footer.jsx
Comment (angelodalisay) [inline src/components/Footer.jsx:1]: overall this looks great!

**Review by angelodalisay — APPROVED** [review]

File: src/components/Footer.jsx (hunk 6 above)
Comment (bjv753) [reply src/components/Footer.jsx:23]: Cool!

**Pull request merged**